src/Compilers/CSharp/Test/Emit2/CodeGen/CodeGenNullCheckedParameterTests.cs

```diff
@@ -2029,5 +2042,209 @@ .maxstack  2
   IL_0024:  ret
 }");
         }
+
+        [Fact]
+        public void AsyncMethod_1()
```

Comment:
consider adding tests for async local function and/or lambda too.

Reply:
Now have tests for both async lambdas and local functions. (I think we did have the local function test before this comment, but good to have the lambda test.)